README.md

```diff
@@ -1,226 +1,187 @@
+o-tracking does not work without JavaScript but, you can still send basic tracking requests to the Spoor API by setting a css background image url which points to the Spoor tracking pixel endpoint (`https://spoor-api.ft.com/px.gif`). Click events and other interactive events will not be tracked without JavaScript but basic page view events can be tracked.
 
-Checkout the [Origami spec](http://origami.ft.com/docs/developer-guide/using-modules/#core-vs-enhanced-experience) for the correct CTM logic, [html and css changes](http://origami.ft.com/docs/developer-guide/using-modules/#styles-for-fallbacks-and-enhancements) needed.
+The Spoor tracking pixel endpoint takes a `data` query parameter which is a url-encoded JSON string that represents the data to track in Spoor.
 
+```html
+<div class="o-tracking o--if-no-js" data-o-component="o-tracking">
+	<div style="background: url('https://spoor-api.ft.com/px.gif?data=YOUR_URL_ENCODED_JSON_DATA_HERE');"></div>
+</div>
+```
 
+For example, if you have the following data you want to track in Spoor:
 
-o-tracking should have the following piece of html added, with the correct server and data filled in.
+```json
+{
+  "category": "page",
+  "action": "view",
+  "system": {
+    "apiKey": "qUb9maKfKbtpRsdp0p2J7uWxRPGJEP",
+    "source": "o-tracking",
+    "version": "1.0.0"
+  },
+  "context": {
+    "product": "ft.com",
+    "content": {
+      "asset_type": "page"
+    }
+  }
+}
+```
+Here is the corresponding tracking pixel setup:
 ```html
 <div class="o-tracking o--if-no-js" data-o-component="o-tracking">
-	<div style="background:url('http://server?data={}');"></div>
+    <div style="background: url('https://spoor-api.ft.com/px.gif?data=%7B%22category%22:%22page%22,%20%22action%22:%22view%22,%20%22system%22:%7B%22apiKey%22:%22qUb9maKfKbtpRsdp0p2J7uWxRPGJEP%22,%22source%22:%22o-tracking%22,%22version%22:%221.0.0%22%7D,%22context%22:%7B%22product%22:%22ft.com%22,%22content%22:%7B%22asset_type%22:%22page%22%7D%7D%7D');"></div>
 </div>
 ```
 
-### Recommended implementation using the build service
+### Tracking with JavaScript
 
-Use an onload handler to check when o-tracking has loaded and then init.
+To manually instantiate `o-tracking`, import the component and call the `init` method with configuration which is specific to your product:
 
-```js
-// CTM
-if (cutsTheMustard) {
-    var o = document.createElement('script');
-    o.async = o.defer = true;
-    o.src = 'https://build.origami.ft.com/v2/bundles/js?modules=o-tracking';
-    var s = document.getElementsByTagName('script')[0];
-    if (o.hasOwnProperty('onreadystatechange')) {
-        o.onreadystatechange = function() {
-            if (o.readyState === "loaded") {
-                oTrackinginit();
-            }
-        };
-    } else {
-        o.onload = oTrackinginit;
+If o-tracking is running on a page which has an ftsession cookie, it will automatically add that information to the data it is tracking.
```

Comment:
This line feels a little out of place between the `init` instructions above and the code example below. I'm not sure if it belongs as part of the previous `init` paragraph or somewhere else?

Reply:
Yeah, I think you are right, it needs to be separate from `init`.